zeppelin/cli/execute.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import os
+import argparse
+import json
+import sys
+from ..executors.notebook_executor import NotebookExecutor
+
+
+def main():
+    """Entry point.
+
+    - Execute notebook
+    - Save output to either file or display it in stderr
+    - Display errors during the run if they exist
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', dest='path_to_notebook_json', required=True,
+                        help='Zeppelin notebook input file (.json)')
+    parser.add_argument('-o', dest='output_path',
+                        help='Path to save rendered output file (.json) (optional)')
+    parser.add_argument('--clean', dest='clean', action='store_true',
```

Comment:
The name of this argument is confusing and it's not clear how it interacts with the output_path parameter. Let's get rid of this and instead use output_path for it. You could set the default value of output_path to sys.stdout.